docs/tutorial_basics.rst

```diff
@@ -119,6 +119,17 @@ Let's inspect what files ReFrame produced for this test:
 ReFrame stores in the output directory of the test the build and run scripts it generated for building and running the code along with their standard output and error.
 All these files are prefixed with ``rfm_``.
 
+   .. tip::
+
+      .. versionadded:: 4.1.0
+
+        Until you get familiar with all the test attributes and options of ReFrame you can run a test in dry-run mode, with the :attr:`~reframe.core.pipeline.RegressionTest.dry_run_mode` attribute.
+        When this attribute is set to `True`, ReFrame will create the build and run script for the test but will not actually submit to the scheduler.
+        You can then find the created job script in the output directory of the test.
+
+        It is recommended to set this attribute directly in the cli with the :option:`--setvar` option instead of marking a test in the test file.
+        Just try adding `-Sdry_run_mode=1` to your command.
```

Comment:
I suggest providing an actual `--dry-run` option that would set a special `__rfm_dry_run__` variable inside the test.